1_html-css/contact.html

```diff
@@ -0,0 +1,35 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Best website ever</title>
+  </head>
+  <body>
+    <div class="menu flex">
+      <div>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="contact.html">Contact</a></li>
+          <li><a href="skills.html">Skills</a></li>
+          <li><a href="portfolio.html">Portfolio</a></li>
+        </ul>
+      </div>
+      <div>
+        <img src="./linkedin-icon.png" />
+      </div>
+    </div>
+    <div class="name"></div>
+    <section>
+      <div>
+        <h3>You can contact me via:</h3>
+        <h2>email m.marciniak89@gmail.com</h2>
+        <h2>https://www.linkedin.com/in/monika-marciniak2/</h2>
```

Comment:
If you want this elements to link somewhere and be clickable, you need to wrap the address in an anchor tag (<a></a>). It also might not be a best idea to put links inside the headings tags: due to accessibility it is probably best to leave the h tags for headings and just style the anchor tags how you want them to look. The structure of your html determines how the webpage is read by screen readers thats people with poor eyesight may use to read the text, and for the screen reader it is important that elements are used correctly. This is of course not that important with exercises but it's good to have that in mind in the future, to make your work accessible for everyone.